optimum/bettertransformer/transformation.py

```diff
@@ -79,6 +79,31 @@ def replace_to_bettertransformer(model, config):
     return model
 
 
+def invert_to_old_model(bt_model):
```

Comment:
Not a big fan of the name of the function, maybe `revert_to_original_model`?

Reply:
fixed in [8ee8df5](https://github.com/huggingface/optimum/pull/676/commits/8ee8df53d6685110ca2db78b94e9b0c0f49d533e)